_data/translations/principles.yml

```diff
@@ -0,0 +1,39 @@
+title:
+  en: "The Paris Call of 12 November 2018"
+  fr: "L'Appel de Paris du 12 novembre 2018"
+subtitle:
+  en: '<a href="#full-text">Full text</a> and <a href="#principles">the 9 principles</a>'
+  fr: '<a href="#full-text">Texte intégral</a> et <a href="#principles">les 9 principes</a>'
+principles-section-title:
+  en: "The 9 principles"
+  fr: "Les 9 principes"
+principle-card-label:
+  en: "Principle"
+  fr: "Principe"
+principle-1:
+  en: "Increase prevention and resilience to malicious online activities"
+  fr: "Accroître la prévention et la résilience face aux activités malicieuses en ligne"
+principle-2:
+  en: "Protecting the accessibility and integrity of the Internet"
+  fr: "Protéger l’accessibilité et l’intégrité d’Internet"
+principle-3:
+  en: "Prevent interference with electoral processes"
+  fr: "Prévenir les interférences aux processus électoraux"
+principle-4:
+  en: "Prevent intellectual property infringement"
+  fr: "Empêcher les violations de la propriété intellectuelle"
+principle-5:
+  en: "Strengthen the fight against the proliferation of malicious tools and computer practices designed to harm"
+  fr: "Renforcer la lutte contre la prolifération d’outils malveillants et de pratiques informatiques destinés à nuire"
+principle-6:
+  en: "Increase the security of digital products and services"
+  fr: "Accroître la sécurité des produits et services numériques"
+principle-7:
+  en: "Take action against cyber-mercenarism and the offensive actions of non-state actors"
+  fr: "Prendre des mesures contre le cyber-mercenariat et les actions offensives des acteurs non-étatiques"
+principle-8:
+  en: "Encourage computer hygiene for all"
+  fr: "Encourager l’hygiène informatique pour tous"
+principle-9:
+  en: "Work together to strengthen relevant international standards"
+  fr: "Travailler ensemble pour renforcer les normes internationales pertinentes"
```

Comment:
Did you take these names from the call translation?

Reply:
Nope